src/pybind/mgr/dashboard/frontend/src/app/shared/datatable/table/table.component.html

```diff
@@ -265,9 +265,11 @@
              let-row="data.row"
              let-value="data.value">
   <div class="position-relative">
+  <div class="sparkline-css">
```

Comment:
why new div ?
```suggestion
```
Also remove the bootstrap class above `position-relative` and can you check if thats what causing issue ?